modules/core/src/main/java/org/locationtech/jts/io/WKBWriter.java

```diff
@@ -272,13 +274,27 @@ public WKBWriter(int outputDimension, int byteOrder) {
   
   /**
    * Creates a writer that writes {@link Geometry}s with
-   * the given dimension (2 or 3) for output coordinates
+   * the given dimension (2 or 4) for output coordinates
```

Comment:
Shouldn't this be `2 to 4`?

Reply:
Correct.

Reply:
Fixed in https://github.com/locationtech/jts/pull/734/commits/268b43691cf2bef399e2f027d669b66bf1fc100c